ara/templates/task_result.html

```diff
@@ -1,5 +1,36 @@
 {% extends "layout.html" %}
 
+{% macro display_result(result, hdr) %}
+
+{% if result is mapping %}
+{% if 'item' in result %}
+<{{hdr}}>Item</{{hdr}}>
+
+<pre>{{result.item|to_nice_json}}</pre>
+{% endif %}
+
+{% for attr in ['msg', 'cmd', 'stdout', 'stderr'] %}
```

Comment:
We need to format the "results" field, that's where the interesting stuff happens and ideally it'd show like the stderr/stdout fields (no \n all over the place madness) ![Alt text](http://i.imgur.com/eiIw8Fm.png)